fluentui_persona/src/main/java/com/microsoft/fluentui/persona/PersonaChipView.kt

```diff
@@ -214,6 +223,10 @@ class PersonaChipView : TemplateView {
             avatarImageBitmap = this@PersonaChipView.avatarImageBitmap
             avatarImageUri = this@PersonaChipView.avatarImageUri
             avatarBackgroundColor  = this@PersonaChipView.avatarBackgroundColor
+            if(this@PersonaChipView.avatarContentDescriptionLabel.isNotEmpty()) {
```

Comment:
should we be allowed to set it to empty also?

Reply:
We will. But since it will not be focusable if not empty. it does not actually make sense but we will remove this extra check

Reply:
thanks, it makes sense that it will not be focusable when empty, that sounds good!